src/Commands/DefaultShooter.h

```diff
@@ -0,0 +1,18 @@
+#ifndef DefaultShooter_H
+#define DefaultShooter_H
+
+#include "Commands/Command.h"
+
+class DefaultShooter : public frc::Command {
+public:
+	DefaultShooter();
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+private:
+
+};
+
+#endif  // DefaultShooter_H
```

Comment:
This is common with header files. This helps to match up the `#endif` with the `#if`.